### PR TITLE
Merge branch 'release'

### PR DESCRIPTION
**Pull request by hluk**

### CODE REVIEW

File: pdc-contact-browser.spec [hunk 1]
```diff
@@ -28,6 +28,10 @@ cp serversetting.json %{buildroot}/var/www/html/%{name}
 /var/www/html/%{name}
 
 %changelog
+* Thu Dec 22 2016 Chuang Zhang <chuzhang@redhat.com> 1.2.0-1
+- new package built with tito
+- fix bugs
+- add assets for packaging
```
Comment (lubomir) [inline pdc-contact-browser.spec:28]: Ideally there should be an empty line here to be consistent with rest of the changelog

**Review by lubomir — CHANGES_REQUESTED** [review]
Do we really want to commit the generated `assets/` directory?

**Comment by hluk** [comment]
> Do we really want to commit the generated assets/ directory?

Oh, if it's generated directory, then no, ideally it shouldn't be in the repo.

Since this is simple merge commit, I'll create another PR for release branch.

**Comment by lubomir** [comment]
I believe it is created by `npm run build:webpack`

**Comment by hluk** [comment]
@lubomir This merge now contains the fixes from PR #66.

**Review by lubomir — APPROVED** [review]

**Pull request merged**